docs/Documentation.md

```diff
@@ -550,7 +550,7 @@ const MyComponent = () => {
 };
 ```
 
-#### useTrackPlayerProgress
+#### usePlaybackState
```

Comment:
useProgress instead of usePlaybackState

Reply:
As @benjipott alreay pointed out, `useProgress` and `usePlaybackState` are completely different hooks.

Reply:
Whoops! Sorry, my bad.